styles.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: #eee;
+}
+nav {
+  width: 100%;
+  height: auto;
+}
+.navbar-container {
+  float: right;
+  width: 300px;
+  display: flex;
+  text-align: right;
+}
+.navbar-button {
+  margin: auto;
+}
+.navbar-button button {
+  background-color: blue;
+  color: white;
+  border: 1px solid black;
+  padding: 10px;
+  border-radius: 8px;
+}
+header {
+  text-align: center;
+  padding-top: 80px;
+  clear: both;
+}
+#headerBrands {
+  width: 380px;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+header img {
+  height: 20px;
+  width: 20px;
+}
+header a {
+  margin: 0 4px 0 4px;
+}
+.form-container {
+  border: 1px solid #333;
+  text-align: center;
+  width: 380px;
+  margin: 0 auto 50px auto;
+  clear: both;
+  border-radius: 2px;
+  background-color: #fff;
+}
+form * {
```

Comment:
Suggestion: Never use `*` in most scenarios, here it'll work without `*`.

Reply:
Hi, actually in this scenario it won't work, since I'm setting display block for all form elements (form *).

Reply:
Ah Ok, didn't check for that one. But try not to use `*` because of performance issues. More about this [here](https://stackoverflow.com/questions/1714096/is-the-css-star-selector-considered-harmful-and-why).

Reply:
Thanks for the reference!